reference/services/fetch-service.rst

```diff
@@ -0,0 +1,149 @@
+
+5. Launchpad makes a final request to the fetch service control endpoint to
+   end session ``session ID``, which deletes all data for that session that
+   was stored in the fetch service database.
+
+Using the fetch service
+~~~~~~~~~~~~~~~~~~~~~~~
+Currently, only snaps can use the fetch service.
+
+To do so, a Launchpad admin is required to set the
+``use_fetch_service`` flag within a snap to ``true``, either in the API or in
+the UI by accessing the Admin area of a snap.
+
+Requests to use the fetch service shall be made to the Launchpad team through
+the usual channels (by
+`opening a question <https://answers.launchpad.net/launchpad>`_, or emailing
+the team at feedback@launchpad.net).
+
+Fetch service maintainers
+~~~~~~~~~~~~~~~~~~~~~~~~~
+The fetch service code itself is not maintained by Launchpad - it is
+maintained by Canonical's Starcraft team. Nonetheless, the service was created
+to be used by Launchpad builders, and that's currently its purpose.
+
+Launchpad maintains its mojo specs and deployment to its environments.
+
+Documentation
+-------------
+* `Snap docs <https://github.com/canonical/fetch-service/tree/main/docs>`_
+
+Git repositories
+----------------
+* `Fetch service Snap <https://github.com/canonical/fetch-service>`_
+* `Fetch service Charm <https://github.com/canonical/fetch-operator>`_
+* `lp-fetch-service Mojo spec <https://git.launchpad.net/~launchpad/launchpad-mojo-specs/+git/private/tree/lp-fetch-service>`_
+
+The following repositories also contain code relevant to how Launchpad uses
+the fetch service:
+
+* `buildd <https://git.launchpad.net/~launchpad/launchpad-buildd>`_
+* `launchpad-buildd-manager <https://git.launchpad.net/~launchpad/launchpad/tree/charm/launchpad-buildd-manager>`_
+
+Bug trackers
+------------
+* `Fetch service snap GitHub issues <https://github.com/canonical/fetch-service/issues>`_
+* `Fetch service charm GitHub issues <https://github.com/canonical/fetch-operator/issues>`_
+* `Launchpad buildd bug tracker <https://bugs.launchpad.net/launchpad-buildd>`_
+
+Deployment
+----------
+We deploy the fetch service using the specs defined in
+`fetch service mojo specs <https://git.launchpad.net/~launchpad/launchpad-mojo-specs/+git/private/+ref/master>`_.
+
+Qastaging
+~~~~~~~~~
+For qastaging deployment, SSH into
+``stg-lp-fetch-service-qastaging@launchpad-bastion-ps5``, and run
+``upgrade-qastaging``.
+
+Production
+~~~~~~~~~~
+For production deployment, SSH into
+``stg-lp-fetch-service@launchpad-bastion-ps5``, and run
+``upgrade-production``.
+
+
+Related specifications
+----------------------
+* `LP136 - Integrating the fetch service for snap builds <https://docs.google.com/document/d/1Z2kVh8eGzDV1-zEyTRYbCNQ0fsXJWt9-vutAjZ9Cxck>`_
+* `ST108 - Fetch service control API <https://docs.google.com/document/d/1Ta0THOsHLwbOA6H7ewHa-6s2GtZRWxvvtiMKFk5jiq8>`_
+
+Log files
+---------
+Production
+~~~~~~~~~~
+Not set up.
```

Comment:
Would IS be able to view the logs? If so, how?

Reply:
That's part of another ticket. I added details here, and added a comment in the ticket to update this section of the documentation